src/dataframe/dataframe.jl

```diff
@@ -721,11 +721,29 @@ Base.hcat(df1::DataFrame, df2::AbstractDataFrame, dfn::AbstractDataFrame...) = h
 ## Missing values support
 ##
 ##############################################################################
+"""
+    allowmissing!(df::DataFrame)
+
+Convert all columns of a `DataFrame` from element type `T` to
+`Union{T, Missing}` to support missing values.
+
+    allowmissing!(df::DataFrame, col::ColumnIndex)
```

Comment:
`ColumnIndex` is not exported, so I guess we should use `Union{Integer, Symbol}`. Not sure what we do in other docstrings.